src/script/components/MessagesList/Message/FailedToAddUsersMessage.tsx

```diff
@@ -101,15 +108,28 @@ const FailedToAddUsersMessage: React.FC<FailedToAddUsersMessageProps> = ({
           data-uie-name="element-message-failed-to-add-users"
           data-uie-value={total <= 1 ? '1-user-not-added' : 'multi-users-not-added'}
         >
-          <p
-            css={warning}
-            dangerouslySetInnerHTML={{
-              __html:
-                total <= 1
-                  ? t('failedToAddParticipant', {name: users[0].name(), domain: users[0].domain})
-                  : t('failedToAddParticipants', {total: total.toString()}),
-            }}
-          />
+          {total <= 1 && (
+            <p data-uie-name="1-user-not-added-details" data-uie-value={users[0].id}>
+              <span
+                css={warning}
+                dangerouslySetInnerHTML={{
+                  __html: t(`failedToAddParticipantSingularOfflineBackEnd`, {
+                    name: users[0].name(),
+                    domain: users[0].domain,
+                  }),
+                }}
+              />
+              {learnMore}
+            </p>
+          )}
+          {total > 1 && (
+            <p
+              css={warning}
+              dangerouslySetInnerHTML={{
+                __html: t(`failedToAddParticipantsPlural`, {total: total.toString()}),
+              }}
+            />
+          )}
```

Comment:
Could be simplified to just `total <= 1 ? () : ()`, no?

Reply:
It could, I kept it the way @thisisamir98  did it.
Probably easier to read?

Reply:
The result will be the same, I'm fine with it